src/query/service/src/servers/http/middleware/session.rs

```diff
@@ -514,5 +450,21 @@ pub async fn json_response<E: Endpoint>(next: E, req: Request) -> PoemResult<Res
         )
             .into_response(),
     };
+    resp.headers_mut()
+        .insert(HEADER_VERSION, QUERY_SEMVER.to_string().parse().unwrap());
```

Comment:
need test to check `HEADER_VERSION` : `QUERY_SEMVER `  is in response.

Reply:
done